doc-content/enterprise-only/integration/dev-studio-plug-in-install-proc.adoc

```diff
@@ -1,24 +1,25 @@
 [id='dev-studio-plug-in-install-proc']
-= Installing Red Hat JBoss Developer Studio plug-ins
+= Installing Red Hat Developer Studio plug-ins
 
-The Red Hat JBoss Developer Studio plug-ins for {PRODUCT}  are based on the community version of the plug-ins. For this reason, the {PRODUCT} plug-in is called the Drools plug-in.
+The Red Hat Developer Studio plug-ins for {PRODUCT} are based on the community version of the plug-ins. For this reason, the {PRODUCT} plug-in is called the Drools plug-in.
 
-//Get the latest Red Hat JBoss Developer Studio from the https://access.redhat.com[Red Hat Customer //Portal]. The {PRODUCT} plug-ins for Red Hat JBoss Developer Studio are available using the update site.
+//Get the latest Red Hat Developer Studio from the https://access.redhat.com[Red Hat Customer //Portal]. The {PRODUCT} plug-ins for Red Hat Developer Studio are available using the update site.
 
 .Procedure
-. Download the {PRODUCT} plug-ins for Red Hat JBoss Developer Studio:
+. Download the {PRODUCT} plug-ins for Red Hat Developer Studio:
 .. Log in to the https://access.redhat.com[Red Hat Customer Portal].
 .. Click *DOWNLOADS* at the top of the page.
-.. On the *Product Downloads* page that opens, navigate to the JBOSS DEVELOPMENT AND MANAGEMENT section, and click *Red Hat JBoss Developer Studio*.
-.. On the *Software Downloads* page, download *Red Hat JBoss Developer Studio Integration Stack 11.3.0 Update Site Zip* (`devstudio-integration-stack-11.3.0.GA-updatesite.zip`).
-//. Unzip the `devstudio-11.3.0.GA-updatesite-core.zip` file.
-. Start Red Hat JBoss Developer Studio by selecting *Red Hat JBoss Developer Studio* from the Start menu.
```

Comment:
Reword:
To start . Start Red Hat JBoss Developer Studio, select *Red Hat JBoss Developer Studio* from the *Start* menu.

Be sure to bold Start, in Start menu. We bold a GUI elements.

Reply:
Done